tests/test_config.py

```diff
@@ -479,7 +498,14 @@ def test_package_manager_tree(tmp_path: Path, skel: Optional[Path], pkgmngr: Opt
     with chdir(tmp_path):
         config = Path("mkosi.conf")
         with config.open("w") as f:
-            f.write("[Content]\n")
+            f.write(textwrap.dedent(
+                """\
+                [Distribution]
+                Distribution=fedora
+
+                [Content]
+                """
+            ))
             if skel is not None:
                 f.write(f"SkeletonTrees={skel}\n")
```

Comment:
If you do this and the same for `pkgmngr` below, you can lose the dedent above.
```suggestion
            if skel is not None:
                f.write(f"""
                SkeletonTrees={skel}
                """
            )
```

Reply:
Isn't the indention of the code a bit weird in this case? The problem is that the if adds a level of indention, so we would need to break formatting convention to indent the string the right level.

Reply:
It the above case it would be the same level of indentation for all the snippets and since the config parser dedents internally, it's fine that all of them are (heavily) indented.